server/configs/application.properties

```diff
@@ -1,15 +1,33 @@
-server.port=8081
+# server.port=8080
 
-labkey.dataSourceName[0]=jdbc/labkeyDataSource
-labkey.driverClassName[0]=@@jdbcDriverClassName@@
-labkey.url[0]=@@jdbcURL@@
-labkey.username[0]=@@jdbcUser@@
-labkey.password[0]=@@jdbcPassword@@
+## To switch to http, uncomment above server config and comment
+## below server config
 
-labkey.masterEncryptionKey=@@masterEncryptionKey@@
-labkey.webAppLocation=@@pathToServer@@/build/deploy/labkeyWebapp
+server.port=8443
+server.ssl.enabled=true
+server.ssl.enabled-protocols=TLSv1.3,TLSv1.2,TLSv1.1
+server.ssl.key-alias=my_selfsigned
+server.ssl.key-store=/path/to/key-store-file
+server.ssl.key-password=pwd
+server.ssl.key-store-type=key-store-type
+server.ssl.ciphers=ciphers
+
+labkeyDataSource.dataSourceName[0]=jdbc/labkeyDataSource
```

Comment:
@labkey-adam Do you have ideas on these property names? I can't decide which I think is best.

I like qualifying the name itself with "labkey" but it's a bit confusing to have all of them include "labkeyDataSource" when the value also needs to be "jdbc/labkeyDataSource"

Reply:
And I'm assuming one can configure other data sources, which wouldn't be named `jdbc/labkeyDataSource`. So you'd have:

```labkeyDataSource.dataSourceName[1]=jdbc/someOtherDataSource```

Which is really confusing.

It looks like other labkey.xml properties (webAppLocation, masterEncryptionKey) are prefaced with `context`. Seems reasonable to be consistent with all properties from the context file, so:

```context.dataSourceName[1]=jdbc/someOtherDataSource```